src/shared/types/assignment.type.ts

```diff
@@ -0,0 +1,18 @@
+type Assignment = {
```

Comment:
Rebase against master and shared will be gone. From there, checkout the new section in the `README` about `devu-shared`. You'll need to clone that repo and create a new branch there with this stuff.

Reply:
If you need help/ have questions about rebasing hit us up and we can help.